modules/connectIdSystem.js

```diff
@@ -127,6 +139,17 @@ function getSiteHostname() {
   return pageInfo.hostname;
 }
 
+function isOAndOTraffic() {
```

Comment:
not covered in tests

Reply:
it is covered by this test
`it('returns an object with the stored ID from cookies and syncs because is O&O traffic', () => {
          const last2Days = Date.now() - (60 * 60 * 24 * 1000 * 2);
          const last21Days = Date.now() - (60 * 60 * 24 * 1000 * 21);
          const ttl = 60 * 60 * 24 * 1000 * 3;
          const cookieData = {connectId: 'foo', he: HASHED_EMAIL, lastSynced: last2Days, puid: '9', lastUsed: last21Days, ttl};
          getCookieStub.withArgs(STORAGE_KEY).returns(JSON.stringify(cookieData));
          const getRefererInfoStub = sinon.stub(refererDetection, 'getRefererInfo');
          getRefererInfoStub.returns({
            ref: 'https://dev.fc.yahoo.com?test'
          });`